flyteplugins/go/tasks/plugins/k8s/spark/spark_test.go

```diff
@@ -853,7 +853,7 @@ func TestBuildResourcePodTemplate(t *testing.T) {
 	assert.Equal(t, defaultConfig.DefaultEnvVars["foo"], findEnvVarByName(sparkApp.Spec.Driver.Env, "foo").Value)
 	assert.Equal(t, defaultConfig.DefaultEnvVars["fooEnv"], findEnvVarByName(sparkApp.Spec.Driver.Env, "fooEnv").Value)
 	assert.Equal(t, findEnvVarByName(dummyEnvVarsWithSecretRef, "SECRET"), findEnvVarByName(sparkApp.Spec.Driver.Env, "SECRET"))
-	assert.Equal(t, 9, len(sparkApp.Spec.Driver.Env))
+	assert.Equal(t, 10, len(sparkApp.Spec.Driver.Env))
```

Comment:
@ppiegaze , @neverett  Looks like I need a review from one of you for the changes in this file. Would appreciate if you can take a quick look.

Reply:
I'll go ahead and approve, but it's strange that GitHub added us to this PR -- we're only listed as codeowners for the /docs/ directory. I'll try to fix that so we don't end up blocking future PRs.